cli/integration_tests/persistent_dependencies/persistent-1-topological.t

```diff
@@ -0,0 +1,21 @@
+# Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd) 1-topological
+
+// Workspace Graph
+// - app-a depends on pkg-a
+// 
+// Make this Task Graph:
+// dev
+// └── ^dev
+//
+// With this workspace graph, that means:
+//
+// app-a#dev
+// └── pkg-a#dev
```

Comment:
Not really. I guess it helps if you don't have syntax highlighting for the files, but I think it was mostly just a habit thing from me. 

The discrepancy of `#` and `//` is because I was copy pasting comments from Go and I guess I didn't make all of them consistent.